ArduPlane/GCS_Mavlink.cpp

```diff
@@ -1213,7 +1213,7 @@ void GCS_MAVLINK_Plane::handleMessage(const mavlink_message_t &msg)
         Location new_home_loc {};
         new_home_loc.lat = packet.latitude;
         new_home_loc.lng = packet.longitude;
-        new_home_loc.alt = packet.altitude / 10;
+        new_home_loc.alt = packet.altitude * 0.1f;
```

Comment:
This is wrong, it was an integer before, this is forcing a promotion to floating point type, then casting back down to an integer.